lib/resources/dstu2/condition.yaml

```diff
@@ -71,6 +72,15 @@ fields:
                    "display": "Generalized nonconvulsive epilepsy",
                    "userSelected": true}],
                 "text": "Absence seizures"}}
+  binding:
+    description: A code that defines the condition.
```

Comment:
These descriptions should probably describe the binding more than the value type. In other words, they should be closer to the values in the FHIR terminology table: http://hl7.org/fhir/DSTU2/condition.html#tx. In this case a lot of the terminology descriptions are the same, or nearly the same, as the field descriptions